src/Pages/HomePage/HomePage.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { SideBar } from '../../Components/SideBar/SideBar';
+import { Header } from '../../Components/Header/Header';
```

Comment:
import these components in a separate file and then import from there , this way readability will be improved

Reply:
Barel Export will be done later! Thank you for suggesting